jenkins/bootstrap_test.py

```diff
@@ -1416,10 +1426,16 @@ def CheckBootstrapYaml(self, path, check, suffix='suffix', use_json=False):
             self.assertTrue(job_name)
             self.assertEquals(job_name, real_job.get('job-name'))
 
+    def GetRealBootstrapJob(self, job):
+        key = os.path.splitext(job.strip())[0][3:]
```

Comment:
Ohhhh... the issue here is that from the job I would not know which yaml file to load